src/main/java/uk/gov/hmcts/ccd/domain/service/common/AccessControlService.java

```diff
@@ -329,8 +321,70 @@ private boolean hasCreateAccess(String newFieldName, final List<CaseField> caseF
     }
 
     private boolean valueDifferentAndHasUpdateAccess(JsonNode newData, JsonNode existingData, String newFieldName, final List<CaseField> caseFieldDefinitions, final Set<String> userRoles) {
-        return existingData.get(newFieldName).equals(newData.get(newFieldName))
-            || hasCaseFieldAccess(caseFieldDefinitions, userRoles, CAN_UPDATE, newFieldName);
+        if (existingData.get(newFieldName).equals(newData.get(newFieldName))) {
+            return true;
+        }
+        Optional<FieldType> fieldType = getCaseFieldType(caseFieldDefinitions, newFieldName);
+        if (!fieldType.isPresent()) {
+            return false;
+        }
+        if (!fieldType.get().getType().equals(COLLECTION)) {
+            return hasCaseFieldAccess(caseFieldDefinitions, userRoles, CAN_UPDATE, newFieldName);
+        } else {
+            if (!itemAddedAndHasCreateAccess(newData, newFieldName, caseFieldDefinitions, userRoles)) {
```

Comment:
Could we extract the block:
```
        if (!itemAddedAndHasCreateAccess(newData, newFieldName, caseFieldDefinitions, userRoles)) {
            return false;
        }
        if (!itemDeletedAndHasDeleteAccess(existingData, newData, newFieldName, caseFieldDefinitions, userRoles)) {
            return false;
        }
        return itemUpdatedAndHasUpdateAccess(existingData, newData, newFieldName, caseFieldDefinitions, userRoles);
```
into a separate method for instance `hasAccessForCollectionItemAction`?